app/TransactionsToFireflySender.php

```diff
@@ -57,6 +61,10 @@ public static function transform_transaction_to_firefly_request_body(
             $description = $transaction->getBookingText();
         }
 
+        if($regex_match !== "" && $regex_replace !== "") {
+            $description = preg_replace($regex_match, $regex_replace, $description);
```

Comment:
I think one should check the return value of preg_replace for null here and, if it is null, crash. I would guess that otherwise, it will crash somewhere later on which makes it harder to debug.